### PR TITLE
add run-hw target for easy hardware-based tests

### PR DESCRIPTION
**Pull request by daringer**

### CODE REVIEW

**Comment by robin-nitrokey** [comment]
Okay!  You could also just pass it through directly to the test suite and change `UsbDevice.find` to accept a list of UUIDs instead of a single UUID.  That would be more general.

https://github.com/Nitrokey/nitrokey-3-tests/blob/79ee739a4870f4b495d1916c8e404fb13399f805/conftest.py#L63
https://github.com/Nitrokey/nitrokey-3-tests/blob/79ee739a4870f4b495d1916c8e404fb13399f805/utils/device.py#L167

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,13 @@ TAG := nitrokey-3-tests
 DOCKER ?= docker
 VENV ?= venv
 
+# put UUIDs which shall be allowed for testing into
+# ALLOWED_UUIDS inside 'variables.mk'
+# (seperate them by whitespace)
+-include variables.mk
+
+ALLOWED_UUIDS ?= NONE
```
Comment (robin-nitrokey) [inline Makefile:13]: Does this line have any effect?

**Review by robin-nitrokey — CHANGES_REQUESTED** [review]
LGTM, but please add `variables.mk` to `.gitignore`.

File: Makefile (hunk 1 above)
Comment (daringer) [reply Makefile]: oopsy, debug artifact, removed

**Review by robin-nitrokey — APPROVED** [review]
Thanks!  Final request:  Please update the “Device selection” section in the README with the changed UUID flag and the new make target.

**Comment by daringer** [comment]
mmmmh, docs, how about this one: 
https://github.com/Nitrokey/nitrokey-3-tests/pull/21/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R71-R78

**Pull request merged**